net-analyzer/nuttcp/files/nuttcp.initd

```diff
@@ -0,0 +1,17 @@
+#!/sbin/openrc-run
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+command="/usr/bin/nuttcp"
+command_background="yes"
+pidfile="/run/${SVCNAME}.pid"
+command_args="${NUTTCP_OPTS}"
+
+depend() {
+	need net
+	provide nuttcp
```

Comment:
You don't `provide` yourself.

Reply:
How do I suppose to depend on nuttcp init, if I will not provide something in it?

Reply:
It just works™.

Reply:
Ok. Removed.